plugins/rbac-backend/src/service/policy-builder.ts

```diff
@@ -39,30 +38,35 @@ export class PolicyBuilder {
     },
     pluginIdProvider: PluginIdProvider = { getPluginIds: () => [] },
   ): Promise<Router> {
-    let adapter;
     const databaseEnabled = env.config.getOptionalBoolean(
       'permission.rbac.database.enabled',
     );
 
-    const databaseManager = await DatabaseManager.fromConfig(
-      env.config,
-    ).forPlugin('permission');
-    // Database adapter work
+    let dbConfig: Config;
     if (databaseEnabled) {
-      adapter = await new CasbinDBAdapterFactory(
-        env.config,
-        databaseManager,
-      ).createAdapter();
+      dbConfig = env.config;
     } else {
-      adapter = new FileAdapter(
-        resolvePackagePath(
-          '@janus-idp/backstage-plugin-rbac-backend',
-          './model/rbac-policy.csv',
-        ),
-      );
-      env.logger.info('rbac backend plugin uses only file storage');
+      // use in memory sqlite database, data won't be persistent
+      dbConfig = new ConfigReader({
+        backend: {
+          database: {
+            client: 'better-sqlite3',
+            connection: ':memory:',
+          },
+        },
+      });
+
+      env.logger.warn(`rbac backend plugin data won't be persistent`);
     }
```

Comment:
We could possibly go with an even simpler approach of just setting `dbConfig = env.config`. This is because for both options we are just looking at what is set in the config value `backend.database`. And if I am not mistaken, the backend will not start without the database being defined within the app-config. We can then remove the `const databaseEnabled = env.config.getOptionalBoolean('permission.rbac.database.enabled')` above and also our config value in `config.d.ts`. 

I think this could help with any potential confusions where someone set the database to true under `permission.rbac.database.enabled` but is using better-sqlite3. Or whenever they set the database to false and are using postgres.

What do you think?

Reply:
Ok let's make it simplier. Project manager provided the same feedback, so let's drop permission.rbac.database.enabled option.